RadixWallet/Features/AssetTransferFeature/Components/ChooseAccount/ChooseReceivingAccount+View.swift

```diff
@@ -1,141 +1,113 @@
 import ComposableArchitecture
 import SwiftUI
 
-// MARK: - ChooseAccount.View
-extension ChooseReceivingAccount {
-	public struct ViewState: Equatable {
-		var manualAccountAddress: String
-		var manualAccountAddressFocused: Bool
-		var chooseAccounts: ChooseAccounts.State
-		var canSelectOwnAccount: Bool
-		var validateAccountAddress: AccountAddress?
-		var manualAddressHint: Hint?
-
-		init(state: ChooseReceivingAccount.State) {
-			manualAccountAddress = state.manualAccountAddress
-			chooseAccounts = state.chooseAccounts
-			manualAccountAddressFocused = state.manualAccountAddressFocused
-			validateAccountAddress = state.validatedAccountAddress
-
-			manualAddressHint = {
-				guard !state.manualAccountAddressFocused, !state.manualAccountAddress.isEmpty else {
-					return .none
-				}
+extension ChooseReceivingAccount.State {
+	var canSelectOwnAccount: Bool {
+		manualAccountAddress.isEmpty
+	}
 
-				switch state.validateManualAccountAddress() {
-				case .invalid:
-					return .error(L10n.AssetTransfer.ChooseReceivingAccount.invalidAddressError)
-				case .wrongNetwork:
-					return .error(L10n.AssetTransfer.Error.wrongNetwork)
-				case let .valid(validAddress):
-					if state.chooseAccounts.filteredAccounts.contains(where: { $0 == validAddress }) {
-						return .error(L10n.AssetTransfer.ChooseReceivingAccount.alreadyAddedError)
-					}
-					return .none
-				}
+	var manualAddressHint: Hint? {
```

Comment:
What's the difference between `validatedManualAccountAddress` & `canSelectOwnAccount` or `manualAddressHint`? I believe all of them should be on the same file, so that we group state logic in just one extension/file.

Reply:
Right, moved them to the same extension